test/runnativetests.cmd

```diff
@@ -179,7 +179,8 @@ goto :main
 :runtest
   pushd %_TestTempDir%
   call :do nativetests.exe
-  if ERRORLEVEL 1 set _HadFailures=1
+  if %ERRORLEVEL% NEQ 0 set _HadFailures=1
+  echo -- runnativetests.cmd ^>^> nativetests.exe exiting with exit code %_HadFailures%

```

Comment:
Apparently it produces a summary if the command completes successfully. This could be improved to help make it more clear when something went wrong while executing nativetests.exe but that's out of scope of this change.

The log from native tests is printed to the standard output of the build and the nativetests.log file is copied to the build outputs.